wdae/wdae/datasets_api/urls.py

```diff
@@ -8,6 +8,9 @@
 
 
 urlpatterns = [
+    url(r'^/deniedPrompt$',
```

Comment:
this should be snake case
```suggestion
    url(r'^/denied_prompt$',
```